moskito-extensions/moskito-nginx/src/main/java/net/anotheria/moskito/core/stats/impl/RateValueHolder.java

```diff
@@ -0,0 +1,298 @@
+package net.anotheria.moskito.core.stats.impl;
+
+import net.anotheria.moskito.core.stats.Interval;
+
+import java.math.RoundingMode;
+import java.text.DecimalFormat;
+
+/**
+ * Designed to measure values on 'per-second' basis.
+ * Use only with values that are constantly increasing or decreasing but not both.
+ *
+ * @author dzhmud
+ */
+public class RateValueHolder extends AbstractValueHolder {
```

Comment:
The idea was in configurable update period in the plugin. But as plugin is now bound to BuiltInUpdater, I switch back to regular DiffLongValueHolder. 
So, this class is of no use right now.